nemo_skills/pipeline/utils.py

```diff
@@ -416,47 +416,66 @@ def cluster_upload(tunnel: SSHTunnel, local_file: str, remote_dir: str):
     print(f"\nTransfer complete")
 
 
-def get_packager():
+def get_packager(cluster_config, env_vars):
```

Comment:
So the thing is that i want the git root to be modifiable at least by end (so that i can set that env inside __init__.py at root of the repo). But yes i agree we should also make root modifyable by the cluster config. This pr is dependent on nemo run pr as well, so it might take a while to merge